test/functional/specs/desktop/C2581.js

```diff
@@ -23,6 +24,7 @@ test("Regression: Queue requests until we receive an ECID.", async () => {
   await t.click(Selector("#event-button"));
 
   await responseStatus(networkLogger.adobedcEndpointLogs.requests, 200);
+  console.log(getBody(networkLogger.adobedcEndpointLogs.requests[0]));
```

Comment:
Let's remove this console log please.

Reply:
I put this line in as an example for using the getBody utility. @shammowla please remove it when you finish out the test case.